src/connectors/tests/mod.rs

```diff
@@ -16,6 +16,10 @@
 //! Connector testing framework
 //!
 //! ....
+
+// TODO: what is this feature? Do we know what it does?
```

Comment:
it tries to guard those tests being run with every call to `cargo test`, as they take long, spinning up docker and stuff.
They are all subsumed underneath the `integration` feature. See our `Cargo.toml`. You'd need to create a new flag for `clickhouse-integration` and add it to the activated features for `integration`, and finally guard this mod by `clickhouse-integration`.

Reply:
Alright, so `es` stands for *ElasticSearch*. That makes more sense now. Thanks :slightly_smiling_face:

Reply:
This should be fixed with [66fe72f](https://github.com/tremor-rs/tremor-runtime/pull/1538/commits/66fe72f4b8be8f933d9e412f1776980c6b669681)